whatsapp_bot.py

```diff
@@ -43,4 +44,4 @@ def whatsapp():
 
 
 async def start_whatsapp_bot():
-    app.run(debug=True, port=5000)
+    app.run(debug=True, host="0.0.0.0", port=5000)
```

Comment:
Do we want debug=True for folks running in production? Perhaps add some comment about this?